apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -101,20 +135,47 @@ class SettingsCog extends Component {
           title={msg.settings()}
           onClick={this.state.canOpen ? this.open : undefined}
         />
-        <Portal
-          closeOnEsc
-          closeOnOutsideClick
-          isOpened={this.state.open}
+        <PopUpMenu
+          className="settings-cog-menu"
+          targetPoint={this.targetPoint}
+          isOpen={this.state.open}
           beforeClose={this.beforeClose}
         >
-          <SettingsMenu
-            className="settings-cog-menu"
-            targetPoint={this.targetPoint}
-            handleClose={this.close}
-          />
-        </Portal>
+          <ManageAssets onClick={this.manageAssets}/>
+          <ToggleMaker onClick={this.toggleMakerToolkit}/>
+        </PopUpMenu>
+        <ConfirmEnableMakerDialog
+          isOpen={this.state.confirmingEnableMaker}
+          handleConfirm={this.confirmEnableMaker}
+          handleCancel={this.hideConfirmation}
+        />
       </span>
     );
   }
 }
 export default Radium(SettingsCog);
+
+export function ManageAssets(props) {
+  return (
+    <PopUpMenu.Item {...props}>
+      {msg.manageAssets()}
+    </PopUpMenu.Item>
+  );
+}
+ManageAssets.propTypes = {
```

Comment:
I was doing this, but `PopUpMenu.Item.propTypes` also requires children, which `ManageAssets` implicitly provides.